addons/project/models/project.py

```diff
@@ -1255,6 +1274,12 @@ def update_date_end(self, stage_id):
             return {'date_end': fields.Datetime.now()}
         return {'date_end': False}
 
+    def _update_analytic_account_id(self, vals):
+        # display_analytic_account_id change: update analytic_account_id
+        if vals.get('display_analytic_account_id') and self.project_analytic_account_id.id != vals.get('display_analytic_account_id'):
+            return {'analytic_account_id': vals.get('display_analytic_account_id')}
+        return {}
```

Comment:
I tested it using an inverse method, the thing is, if you changes the project_id.analytic_account_id in an other tab, while having your task form opened, when you save the record, in the inverse method the project_analytic_account_id has already the new value so it could be different than display_analytic_account_id, while display_analytic_account_id hasn't been manually changed.

But finally I asked AJU if I had to do all this """magic""" and she said I only had to have a default value for analytic account (like it was the case before this fixup).

Reply:
Specs changed. Nevermind, thanks for your help !